README.md

```diff
@@ -1,12 +1,9 @@
 # Rancher Logging Examples
 
-This repository contains 2 example charts for testing your logging setup and a simple application for generating logs.
+This repository contains an example charts for testing your logging setup.
 
 ## Charts
 
-### Simple Application
-The `simple-application` chart will deploy 2 pods that will constantly out logs.  This chart is recommended for when you want to configure your own logging setup and need a simple application to collect logs from.
-
 ### Rancher Logging Example

```

Comment:
nit: do we want to remove this section?

Reply:
From all the discussions a use case for this chart has not come up and with this update it became behind I don't see us maintaining it as it is redundant to the main chart.